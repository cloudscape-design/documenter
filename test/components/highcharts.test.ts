// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

import { expect, test } from 'vitest';
import { buildProject } from './test-helpers';

test('should not expand highcharts types', () => {
  const result = buildProject('highcharts');
  expect(result).toMatchSnapshot();
});
