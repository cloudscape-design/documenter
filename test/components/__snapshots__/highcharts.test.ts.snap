// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should not expand highcharts types 1`] = `
[
  {
    "dashCaseName": "chart",
    "description": "Chart description",
    "events": [
      {
        "cancelable": false,
        "deprecatedTag": undefined,
        "description": undefined,
        "detailInlineType": {
          "name": "{ point: Highcharts.Point; }",
          "properties": [
            {
              "name": "point",
              "optional": false,
              "type": "Highcharts.Point",
            },
          ],
          "type": "object",
        },
        "detailType": "{ point: Highcharts.Point; }",
        "name": "onHighlight",
        "systemTags": undefined,
      },
    ],
    "functions": [],
    "name": "Chart",
    "properties": [],
    "regions": [],
    "releaseStatus": "stable",
    "systemTags": undefined,
  },
]
`;
